go/nogo.rst

```diff
@@ -31,15 +31,53 @@ compiler-like errors.
 ``nogo`` is a powerful tool for preventing bugs and code anti-patterns early
 in the development process.
 
-.. contents:: :depth: 2
+.. contents:: .
+  :depth: 2
 
 -----
 
-Overview
---------
+Setup
+-----
+
+Create a `nogo`_ target in a ``BUILD`` file in your workspace. The ``deps``
+attribute of this target must contain labels all the analyzers targets that you
+want to run.
+
+.. code:: bzl
+
+    load("@io_bazel_rules_go//go:def.bzl", "nogo")
+
+    nogo(
+        name = "my_nogo",
+        deps = [
+            ":importunsafe",
```

Comment:
Include a `go_tool_library` target in this example, along with a `load`. It's explained below, so I don't think you need to explain its purpose here, but it would be good to preclude people from trying `go_library` first and encountering a confusing error.

To keep this short, consider reducing `deps` to two entries, one defined here and one elsewhere.

Use a real example for the one defined elsewhere. I think we are going to want to write `go_tool_library` targets for everything in `golang.org/x/tools/go/analysis` so those can easily be referenced. I'ved filed #1780 for that. No need to block on the fix, but it would be good to have a runnable example soon.

Reply:
Done.